src/languageservice/serviceclient.ts

```diff
@@ -146,9 +146,43 @@ export default class SqlToolsServiceClient {
          });
     }
 
+    /**
+     * Initializes the SQL language configuration
+     *
+     * @memberOf SqlToolsServiceClient
+     */
+    private initializeLanguageConfiguration(): void {
+        languages.setLanguageConfiguration('sql', {
+            comments: {
+                lineComment: '--',
+                blockComment: ['/*', '*/']
+            },
+
+            brackets: [
```

Comment:
I don't see any change in sql.configuration.json. shouldn't we remove the configurations for brackets there?